src/main/scala/beam/agentsim/agents/vehicles/VehicleEnergy.scala

```diff
@@ -116,7 +116,11 @@ class ConsumptionRateFilterStoreImpl(
       .foreach(csvRecord => {
         val speedInMilesPerHourBin = convertRecordStringToRange(csvRecord.getString(speedBinHeader), isDouble = true)
         val gradePercentBin = convertRecordStringToRange(csvRecord.getString(gradeBinHeader), isDouble = true)
-        val numberOfLanesBin = convertRecordStringToRange(csvRecord.getString(lanesBinHeader))
+        val numberOfLanesBin = if (csvRecord.getMetaData.containsColumn(lanesBinHeader)) {
+          convertRecordStringToRange(csvRecord.getString(lanesBinHeader))
+        } else {
+          convertRecordStringToRange("(0,100]")
```

Comment:
As long as this is valid then I'm fine with it. The biggest question to me is if a lanes header is a required field. I'll approve, but pinging @wrashid and @colinsheppard so they can chime in if needed